plugins/inputs/wireless/README.md

```diff
@@ -9,30 +9,69 @@ The wireless plugin gathers metrics about wireless link quality by reading the `
 [[inputs.wireless]]
   ## Sets 'proc' directory path
   ## If not specified, then default is /proc
+  ## Ignored on macOS/darwin
   # host_proc = "/proc"
 ```
 
+The `host_proc` option is ignored on macOS.
+
 ## Metrics
 
+### Linux metrics
+
 - metric
   - tags:
     - interface (wireless interface)
   - fields:
-    - status (int64, gauge) - Its current state. This is a device dependent information
-    - link (int64, percentage, gauge) - general quality of the reception
-    - level (int64, dBm, gauge) - signal strength at the receiver
-    - noise (int64, dBm, gauge) - silence level (no packet) at the receiver
-    - nwid (int64, packets, counter) - number of discarded packets due to invalid network id
-    - crypt (int64, packets, counter) - number of packet unable to decrypt
-    - frag (int64, packets, counter) - fragmented packets
-    - retry (int64, packets, counter) - cumulative retry counts
-    - misc (int64, packets, counter) - dropped for un-specified reason
-    - missed_beacon (int64, packets, counter) - missed beacon packets
+    - status (int64, metric) - Its current state. This is a device dependent information
+    - link (int64, percentage, metric) - general quality of the reception
+    - level (int64, dBm, metric) - signal strength at the receiver
+    - noise (int64, dBm, metric) - silence level (no packet) at the receiver
+    - nwid (int64, packets, metric) - number of discarded packets due to invalid network id
+    - crypt (int64, packets, metric) - number of packet unable to decrypt
+    - frag (int64, packets, metric) - fragmented packets
+    - retry (int64, packets, metric) - cumulative retry counts
+    - misc (int64, packets, metric) - dropped for un-specified reason
+    - missed_beacon (int64, packets, metric) - missed beacon packets
```

Comment:
Why removing useful information and replacing with something that doesn't say anything? (`counter` vs `metric`)

Reply:
That change is a mystery. I do not recall making those changes. I will of course immediately revert them.

Reply:
I will check in the change once the other matter is decided.